scripts/remove

```diff
@@ -19,20 +20,55 @@ domain=$(ynh_app_setting_get --app=$app --key=domain)
 final_path=$(ynh_app_setting_get --app=$app --key=final_path)
 
 #=================================================
-# REMOVE THE MAIN DIR OF THE APP
+# STANDARD REMOVE
+#=================================================
+# ...
 #=================================================
-ynh_script_progression --message="Removing $app main directory..." --weight=3
 
-# Remove sources
+#=================================================
+# REMOVE APP MAIN DIR
+#=================================================
+ynh_script_progression --message="Removing app main directory..." --weight=1
+
+# Remove the app directory securely
 ynh_secure_remove --file="$final_path"
```

Comment:
We should delete `$final_path/_h5ai` and check for `--purge` to delete `$final_path`

Reply:
```suggestion
# Remove the app directory securely
ynh_secure_remove --file="$final_path/_h5ai"
```